react/lib/util/constants.ts

```
@@ -20,10 +20,15 @@ export type ButtonSize = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | "extrasmall" | "smal
+export const DEFAULT_MINIMUM_DONATE_AMOUNT = 10;
+
+export const DONATION_RATE_STORAGE_KEY = 'paybutton_donation_rate'
 export const DONATION_RATE_FIAT_THRESHOLD = 5;
 
 
 export const DEFAULT_MINIMUM_DONATION_AMOUNT: { [key: string]: number } = {
     BCH: 0.00001000,
     XEC: 10,
 };
+
+export const DONATION_RATE_STORAGE_KEY = 'paybutton_donation_rate'
```

Biome (2.1.2)

[error] Shouldn't redeclare 'DONATION_RATE_STORAGE_KEY'. Consider to delete it or rename it.

'DONATION_RATE_STORAGE_KEY' is defined here:


(lint/suspicious/noRedeclare)